Sming/SmingCore/Timer.cpp

```diff
@@ -182,15 +181,23 @@ void Timer::processing(void *arg)
 					ptimer->stop();
 			}
 		}
-
-		if (ptimer->callback)
-		{
-			ptimer->callback();
-		}
-		else if (ptimer->delegate_func)
-		{
-			ptimer->delegate_func();
-		}
+      ptimer->loop();
 	}
 
 }
+
+
+void Timer::loop()
```

Comment:
@it-guru Please, rename the 'loop' method to 'tick'. Otherwise the Arduino people will be confused with the purpose of that one.